models/src_reddit_ads.yml

```diff
@@ -17,7 +17,7 @@ sources:
     
     tables:
       - name: account
-        identifier: "{{ var('reddit_ads_account_identifier', 'account ') }}"
+        identifier: "{{ var('reddit_ads_account_identifier', 'account') }}"
```

Comment:
This extra space was causing my initial compile stage to fail. We will need to remove it in order to ensure proper runs.

Reply:
yeah i noticed that earlier today, this has been updated since